components/net/image_cache.rs

```diff
@@ -41,6 +41,15 @@ use crate::resource_thread::CoreResourceThreadPool;
 // something in higher resolution.
 const FALLBACK_RIPPY: &[u8] = include_bytes!("../../resources/rippy.png");
 
+// The current SVG stack relies on `resvg` to provide the natural dimensions of
+// the SVG, which it automatically infers from the width/height/viewBox properties
+// of the SVG. Since these can be arbitrarily large, this can cause us to allocate
+// a pixmap with very large dimensions leading to the process being killed due to
+// memory exhaustion. For example, the `/css/css-transforms/perspective-svg-001.html`
+// test uses very large values for viewBox. Hence, we just clamp the maximum
+// width/height of the pixmap allocated for rasterization.
```

Comment:
Your rustdoc is missing a slash here:

```suggestion
/// The current SVG stack relies on `resvg` to provide the natural dimensions of
/// the SVG, which it automatically infers from the width/height/viewBox properties
/// of the SVG. Since these can be arbitrarily large, this can cause us to allocate
/// a pixmap with very large dimensions leading to the process being killed due to
/// memory exhaustion. For example, the `/css/css-transforms/perspective-svg-001.html`
/// test uses very large values for viewBox. Hence, we just clamp the maximum
/// width/height of the pixmap allocated for rasterization.
```

Reply:
Oops, addressed in [43e2c08](https://github.com/servo/servo/commit/43e2c08d9fb463ddfb9d8ac11ef473d87ab66ad1).